app/controller.js

```diff
@@ -128,6 +137,11 @@ class MainController {
                 self.undo();
             } else if (e.key === 'Enter') {
                 self.playRegion();
+            } else if (isDownCtrl) {
```

Comment:
Why does this shortcut here? Isn't Ctrl+leftarrow enough?

Reply:
It's needed here because Omri said to add back 5 secs shortcut for all modes